src/screen/AccountsScreen.tsx

```diff
@@ -38,104 +58,77 @@ export function AccountsScreen({navigation}: {navigation: NavigationProp<Complet
   const [sortMenuVisible, setSortMenuVisible] = React.useState(false);
 
   return (
-    <SafeView edges={noTopEdges}>
-      {isLoading ? (
-        <LoadingView />
-      ) : (
-        <FlatList
-          style={styles.container}
-          contentContainerStyle={styles.content}
-          data={combinedData?.sort(sortByFunction)}
-          showsVerticalScrollIndicator
-          keyExtractor={(item) => item.account.address}
-          renderItem={({item}) => (
-            <AccountItem
-              isExternal={item.account.isExternal}
-              identity={item.identity}
-              isFavorite={item.account.meta.isFavorite}
-              toggleFavorite={() => toggleFavorite(item.account.address)}
-              onPress={() => {
-                navigation.navigate(myAccountScreen, {address: item.account.address});
-              }}
-            />
-          )}
-          ItemSeparatorComponent={() => <Divider />}
-          ListEmptyComponent={() => (
-            <View style={styles.emptyContainer}>
-              <Image source={require('src/image/no_accounts.png')} style={styles.emptyImage} />
-              <Padder scale={1} />
-              <Text style={styles.emptyText} status="basic" category="h4">
-                No accounts added
-              </Text>
-              <Padder scale={1.5} />
-              <Text category="c1">Please add an account to take further actions</Text>
-            </View>
-          )}
-          ListHeaderComponent={() => (
-            <View style={styles.header}>
-              <OverflowMenu
-                anchor={() => (
-                  <TouchableOpacity
-                    onPress={() => {
-                      setSortMenuVisible(true);
-                    }}
-                    style={globalStyles.rowAlignCenter}>
-                    <Text category="c1">Sort by</Text>
-                    <Padder scale={0.5} />
-                    <Icon
-                      name="arrow-ios-downward-outline"
-                      style={globalStyles.icon}
-                      fill={globalStyles.iconColor.color}
-                    />
-                  </TouchableOpacity>
-                )}
-                placement="bottom end"
-                style={styles.overflowMenu}
-                visible={sortMenuVisible}
-                onSelect={({row}: {row: number}) => {
-                  setSortMenuVisible(false);
-                  switch (row) {
-                    case 0:
-                      setSortBy('name');
-                      break;
-                    case 1:
-                      setSortBy('favorites');
-                      break;
-                  }
+    <Provider>
```

Comment:
It seems we need to pass the theme here so the FAB uses our colors

Reply:
yes, thanks for noticing 👍